MaintenanceApi.yaml

```diff
@@ -555,6 +558,10 @@ components:
           type: integer
```

Comment:
Why does this include the initial registration as a recovery?

Reply:
It's "visits to AVAILABLE state". Maybe this could be worded differently.